homeassistant/components/media_player/const.py

```diff
@@ -51,6 +51,8 @@ class MediaPlayerState(StrEnum):
     PAUSED = "paused"
     STANDBY = "standby"
     BUFFERING = "buffering"
+    UNKNOWN = "unknown"
+    UNAVAILABLE = "unavailable"
```

Comment:
`STATE_UNKNOWN` is used in `group`, `snapcast` and `universal` media player
`STATE_UNKNOWN` is used in `group` and `universal` media player

Reply:
I am not sure if they really need to be documented, or if they should just be behind the scenes (maybe with a comment)

Reply:
These should not be added here. Unavailable state is set via the available property. Unknown state is set via `None`.

Reply:
Thanks for confirming. I'll adjust the components manually.